File: lua/refactoring/tests/refactor/119/go/custom_extract_var_statement/extract_var.expected.go
package main

import (
    "fmt"
    "math"
)

type Order struct {
    quantity  float64
    itemPrice float64
}

func orderCalculation(order Order, a int, b int) (total float64) {
    temp := a * b
    var basePrice float64 = order.quantity*order.itemPrice
    blah := (basePrice) - 7
    fmt.Println(blah, temp)
    return basePrice -
        math.Max(0, order.quantity-500)*order.itemPrice*0.05 +
        math.Min(basePrice*0.1, 100)
}

File: lua/refactoring/tests/refactor/119/java/custom_extract_var_statement/extract_var.expected.java
class Order {
  double quantity;
  double itemPrice;
}

class ExtractVar {
  public static double orderCalculation(Order order, int a, int b) {
    double temp = a * b;
    private var basePrice = order.quantity * order.itemPrice;
    double blah = (basePrice) - 7;

    System.out.printf("%d %d", temp, blah);

    return basePrice -
        Math.max(0.0, order.quantity - 500) * order.itemPrice * 0.05 +
        Math.min(basePrice * 0.1, 100.00);
  }
}
